Project.toml

```diff
@@ -20,6 +20,6 @@ Antic_jll = "e21ec000-9f72-519e-ba6d-10061e575a27"
 AbstractAlgebra = "^0.9.1"
 BinaryProvider = "0.4, 0.5"
 julia = "1"
-LoadFlint = "^0.2.1"
-Arb_jll = "= 2.17.0"
-Antic_jll = "^0.1.0"
+LoadFlint = "= 0.2.1"
+Arb_jll = "= 2.18.0"
+Antic_jll = "= 0.2.1"
```

Comment:
I am confused -- shouldn't this use LoadFlint 0.3.0 (which is not yet released) to get Flint 2.6.0 ?